### PR TITLE
Rails Blog Views

### PR DESCRIPTION
**Pull request by ezraermy**
In this pull request, I:
- Created a view file for each model.
- Used preprocessed HTML file with embedded Ruby code.
- Used layouts and templates for shared content.

### Other requirements 
- No linter errors
- Document professional way
- Follow gitflow

### CODE REVIEW

File: app/views/shared/_comments.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<br>
+<ul class="comments">
+  <% if post.comments.empty? %>
+    <li>There are no comments yet.</li>
+  <% else %>
+    <%= post.comments.each do |comment| %>
+      <%= render partial: 'shared/comment', locals: { comment: comment } %>
+    <% end %>
+  <% end %>
+</ul>
```
Comment (laiifuu) [inline app/views/shared/_comments.html.erb]: There are a couple of issues with this view: 
- The list of comment objects are being rendered after the list of comments that are properly listed: 
image
- The list of comments in the user's `show` view should only list the last 5 comments added to the post ( the result of the recent_comments method in the Post model ). As you can see in the previous image, 8 comments are being shown. 
- The number of comments at the top is not being updated.

File: app/views/shared/_user.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<li class="user">
+  <%= image_tag user.photo %>
+  <div class="blue-box">
+    <h2><%= link_to user.name, user_path(user) %></h2>
+    <p>Number of posts: <%= if user.posts.any? then user.postCounter else 0 end %></p>
```
Comment (laiifuu) [inline app/views/shared/_user.html.erb:5]: The number of posts is not being displayed correctly, I created multiple posts for this user, and yet 0 is being displayed: 
image

**Review by laiifuu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: 

## Hi @ezraermy  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close


## Highlights
- [x] Green linters check :heavy_check_mark: 
- [x] Nice use of partials :+1: 
- [x] The views respect the wireframes given :+1: 

## Required Changes ♻️
_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and :computer: Happy coding :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**